cijoe/workflows/bench-report.yaml

```diff
@@ -0,0 +1,14 @@
+---
+doc: Plot and report
+
+steps:
+- name: plot
+  uses: bench_plotter
+  with:
+    path: /Users/safl/Downloads/cijoe/bench-results
```

Comment:
Is this path correct? Looks very user-specific...

Reply:
Right, the entire file should not be included, dropping it.